pkgs/development/libraries/boost/darwin-1.55-no-system-python.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/tools/build/src/tools/python.jam b/tools/build/src/tools/python.jam
```

Comment:
I think this name would read better if it was called "darwin-no-system-python-1.55.patch"

Reply:
I see this is consistent with naming of other patches in directory like "cygwin-1.57.0-system-cygwin.patch". So it's fine to leave as is.